README.md

```diff
@@ -192,6 +199,9 @@ Custom content with the CSS class `mat-select-search-custom-header-content` can
 </ngx-mat-select-search>
 ```
 
+#### Global options
+Providing the `MATSELECTSEARCH_GLOBAL_OPTIONS` InjectionToken, several options can be set globally.
```

Comment:
a reference to the actual code would be nice here

Reply:
Isn't that just _business as usual_? Would be nice if you'd adopt it to your needs since I'm not sure what you expect here :) Mention all the inputs again is a bit redundant.

Reply:
Yeah I think here we could simply add a link to the symbol in the github repository, e.g. https://github.com/bithost-gmbh/ngx-mat-select-search/blob/master/src/app/mat-select-search/mat-select-search-clear.directive.ts#L13

Reply:
Thinking of it, it would be great to add a small example here, so many people can benefit from your feature!